docs/content/database-vendors.md

```diff
@@ -24,6 +24,7 @@ Here is a matrix of the current state of support:
 | MySQL 5.7 or higher    |  <Badge type="tip" text="✔" title="Working"/> | <Badge type="tip" text="✔" title="Working"/> | <Badge type="tip" text="✔" title="Working"/> |
 | SQLite    |  <Badge type="danger" text="✘" title="Unsupported"/> | <Badge type="warning" text="~" title="Only unit-tested"/> | <Badge type="danger" text="✘" title="Unsupported"/> |
 | Oracle    |  <Badge type="danger" text="✘" title="Unsupported"/> | <Badge type="warning" text="~" title="Only unit-tested"/>  | <Badge type="danger" text="✘" title="Unsupported"/> |
+| SQLite    |  <Badge type="danger" text="✘" title="Unsupported"/> | <Badge type="warning" text="~" title="Only unit-tested"/>  | <Badge type="danger" text="✘" title="Unsupported"/> |
```

Comment:
It was there already ;)

If you modify this part, remove Oracle which should not be there